ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/ChainDB/Impl/Types.hs

```diff
@@ -274,6 +277,7 @@ data ChainDbEnv m blk = CDB
     -- containing it. The fragment is usually anchored at a recent immutable
     -- tip; if it does not, it will conservatively be treated as the empty
     -- fragment anchored in the current immutable tip.
+  , cdbLoEBlocks       :: !(StrictTVar m [Header blk])
```

Comment:
We should document what this is and some properties (it is empty if the LoE is disabled, blocks are only removed by GC, ...).

Reply:
✅